.env.example

```
@@ -35,6 +35,7 @@ AUTH_EMAIL_RESEND_COOLDOWN_SECONDS=60
 AUTH_EMAIL_MAX_ATTEMPTS=5
 AUTH_EMAIL_VERIFIED_TTL_SECONDS=1800
 AUTH_EMAIL_FROM_ADDRESS=
+AUTH_EMAIL_REPLY_TO=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_EMAIL_REPLY_TO key should go before the AUTH_EMAIL_RESEND_COOLDOWN_SECONDS key

(UnorderedKey)